consensus/polybft/contractsapi/gen/main.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"runtime"
+
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/contractsapi/artifact"
+	"github.com/dave/jennifer/jen"
+)
+
+func main() {
```

Comment:
Correct me if I'm wrong. If I am manually compiling contracts on my local machine, I would need to run this after that?